### ISSUE TITLE
update python to fetch python prebuilt binaries from `actions/python-versions`

### ISSUE THREAD

**Issue by joshspicer** [devcontainers/features#75]
We are planning to collaborate with the actions team and their effort to prebuilt binaries (eg https://github.com/actions/python-versions)

Starting with python, teach our feature to first attempt to pull the appropriate binary from this repo, and if that fails fallback to build from src (same behavior as current).

**Comment by Chuxel** [comment]
Just as a FYI - This is some code I wrote to do this install.

https://github.com/Chuxel/devcontainer-features/blob/main/devcontainer-features/features/python/bin/acquire#L16-L37

The thing I didn't do is parse the manifest file they have already to get the available versions.  However, I did do that in the "Devpacks" prototype I wrote in Go.  I generalized the code there so I could reuse it for Node and other things as well.

https://github.com/Chuxel/devpacks/blob/main/devpacks/internal/common/actions/version_manifest.go
...and...
https://github.com/Chuxel/devpacks/blob/main/devpacks/internal/buildpacks/cpython/cpython_builder.go#L100-L119

I think it would be good to generally use the Actions builds where we can since it does reduce the number of total dependencies for these features.

**Comment by samruddhikhandale** [comment]
ℹ️  The prebuilt binaries are available only for these distros 👇 

![image](https://user-images.githubusercontent.com/24955913/211121476-3a988b63-8984-4c0f-8d1a-cc0c4fbe9d0e.png)

In our case, only for ubuntu. Looks like these will only be useful for Ubuntu, but good improvement nonetheless 🤔

**Comment by Chuxel** [comment]
Yes, which is a good start - but I'd suggest connecting with the Actions folks to see if we can increase coverage. Largely the steps are the same to build Debian 10/11, etc. (I believe @jkeech was thinking about whether some of the same folks could get involved here as well.)
